fabric-client-gametest-api-v1/build.gradle

```diff
@@ -0,0 +1,10 @@
+version = getSubprojectVersion(project)
+
+loom {
+	accessWidenerPath = file('src/client/resources/fabric-client-gametest-api-v1.accesswidener')
+}
+
+moduleDependencies(project, [
+	'fabric-api-base',
+	'fabric-resource-loader-v0'
```

Comment:
Question: What is resource-loader used for?

Reply:
I copied it from another module and left it in because I thought maybe I'd be adding some translatable text. But I never ended up doing it